03.bowling/y-bowling.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+split_score = score.split(',')
+shots = []
+
+split_score.each do |s|
+  if shots.length < 18
+    if s == 'X'
+      shots << 10
+      shots << 0
+    else
+      shots << s.to_i
+    end
+  else
+    shots << if s == 'X'
+               10
+             else
+               s.to_i
+             end
+  end
+end
+
+frames = []
+shots.each_slice(2) do |s|
+  frames << s
+end
+
+frames[9] += frames.pop if frames.length == 11
+
+point = 0
+frames.each_with_index do |frame, i|
```

Comment:
```suggestion
point = frames.each_with_index.sum do |frame, i|
```

sumメソッドを使って`point +=`をなくしてみましょう